### ISSUE TITLE
[REC] OUTLAW Skill suggestion only during !stealthed not working

### ISSUE THREAD

**Issue by 182cm82kg22cm** [Hekili/hekili#3193]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [x] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

Basically I don't want Blade FLurry to be recommended when in stealth but only outside of it.
How do I do that?

1.Inside the action list under "conditions" ofblade flurry,  adding !stealthed  doesn't work
2. Inside the profile under "execute before combat" and "execute when actor is available" adding the same code doesn't work either.
(While in the pre combat  stealth is might be not a huge issue...when vanishing, there is a big change it will be recommended right after, and therefore during stealth)

Side request, is there a way to remove the spiral GCD? I saw a post a year ago that it's not intended to be changable..is it now?

### How to Reproduce

Enter the game as rogue with outlaw spec
have a rotation profile that recommends blade flurry
go into stealth

### Snapshot (Link)

https://pastebin.com/CMBZMdQ7

**Comment by chevyn11** [comment]
You have deft maneuvers talented, blade flurry is probably one of your best cp builders.. especially to keep using BTE in crackshot (during subterfuge, after vanish).
Your snapshot has you standing near multiple enemies in stealth, opening with blade flurry is also correct. Both for building combo points and getting blade flurry up in an aoe situation.

**Comment by 182cm82kg22cm** [comment]
Let me rephrase the problem.  Blade Flurry is being recommended during stealth even though it is already buffed, in which case obviously using Ambush or any other opener is more effective after vanishing. 
(new snapshot with the buff on  https://pastebin.com/L4yiVQkx )
Thus, **all I wanna know is HOW to generally turn off Blade flurry during stealth only**, if it is already active..or if easier, regardless of any condition.
I can use it separately.

**Comment by stantsz** [comment]
> obviously using Ambush

Check those assumptions, buddy.  BF as a builder is a 1.4% gain: https://www.raidbots.com/simbot/report/sWkcPcXnTuchdmRNwRFfCF

**Comment by chevyn11** [comment]
Again, its one of your best cp builders with deft maneuvers. Its correct to use it even if its already up just to build cp. Much better than ambush in multi target.

From the rogue discord:
![image](https://github.com/Hekili/hekili/assets/111624332/2c5bafa8-b782-46ac-b526-85902189983d)

You can go into Abilities > look for blade flurry in the drop down and disable it if you want to disable it altogether.

**Comment by 182cm82kg22cm** [comment]
Thank you. Appreciate the care!
